### ISSUE TITLE
How to ignore Iphone mute switch

### ISSUE THREAD

**Issue by MarsCaiWORD** [CreateJS/SoundJS#312]
I want when the iphone mute switch is turned on keep music play
Is there any solution?  btw  i use  web audio api  play music

iphone 6
QQ browser

**Comment by justinborror** [comment]
Yes you got to go to settings and activate the music app to draw over other
apps or settings or you can try to go to volume control an set it from
there it might be easier

**Comment by MarsCaiWORD** [comment]
@justinborror  Thank you for your reply, but I don’t understand what you mean.  "activate the music app" i run  in QQ browser

**Comment by justinborror** [comment]
Ok go to you phone settings an look for a setting that alows you to draw
1app over another then set your browser to do that ovwr the phone muting

**Comment by MarsCaiWORD** [comment]
sor , I still confused do you have Skype?

**Comment by MarsCaiWORD** [comment]
@justinborror   for example  [like this website use web audio api play music ](https://www.createjs.com/tutorials/SoundJS%20and%20PreloadJS/internalPreloading.html)
when i  use QQ browser open this website  and  turn on iphone mute switch ,   is  mute , but i want  unmute

**Comment by banjerluke** [comment]
There is a way to unlock audio from the Web Audio API while the iPhone/iPad ringer switch is off. The trick is to play a short bit of silence from an HTML5 audio tag. This "unlocks" the audio for that browser window and lets the Web Audio API audio come through.

Here's a gist with the code I use:
https://gist.github.com/banjerluke/b8dd69000d9d874d52602e222fdf6646

**Comment by MarsCaiWORD** [comment]
@banjerluke  thank you , it's working ,  you made my day ,
